tests/playwright/sanity/atomic-widgets.test.ts

```diff
@@ -42,11 +47,5 @@ test.describe( 'Atomic Widgets', () => {
 			} );
 		} );
 	} );
-
-	async function enableRequiredExperiments() {
-		await wpAdmin.setExperiments( {
-			atomic_widgets: 'active',
-		} );
-	}
```

Comment:
don't you need to reset the experiments? 
it's the same server, so you might introduce side effects

Reply:
I reset the experiments on line 29 (afterAll)